docs/token_authorization_api.md

```diff
@@ -45,9 +48,7 @@
 ## Customization
 Torchserve offers various ways to customize the token authorization to allow owners to reach the desired result.
```

Comment:
Could we double check that this statement on customizability is still valid. In case this is not intended, we can delete it or test using a plugin to see if `token` implementation provided in the frontend can be overridden by a plugin.

Reply:
Will remove customization section and move time to expiration to notes.  Plugin implementation no longer works